src/nunit.analyzers/Operations/ConstraintExpression.cs

```diff
@@ -1,63 +1,65 @@
 using System.Collections.Generic;
 using System.Linq;
 using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Operations;
 using NUnit.Analyzers.Constants;
 using NUnit.Analyzers.Extensions;
 
-namespace NUnit.Analyzers.Syntax
+namespace NUnit.Analyzers.Operations
 {
     /// <summary>
     /// Represents assert constraint expression, e.g. 'Is.EqualTo(expected)', 'Is.Not.Null & Is.Not.Empty'
     /// </summary>
     internal class ConstraintExpression
     {
```

Comment:
Suggest type rename to ConstraintOperation (similar as you did for test class)

Reply:
I thought about that, but `Expression` is not a Roslyn term here, but NUnit.
See https://github.com/nunit/nunit/blob/master/src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

Reply:
OK, then you should rename the unit test back to ConstraintExpressionTests.

Reply:
Yes, agree